src/LiveComponent/assets/src/Component/plugins/ErrorPlugin.ts

```diff
@@ -0,0 +1,224 @@
+
+        // querySelectorAll doesn't include the element itself
+        if (element.hasAttribute(ErrorPlugin.errorAttribute)) {
+            matchingElements = [element, ...matchingElements];
+        }
+
+        matchingElements.forEach((element) => {
+            if (!(element instanceof HTMLElement) && !(element instanceof SVGElement)) {
+                throw new Error(
+                    `Element "${element.nodeName}" is not supported for ${ErrorPlugin.errorAttribute} directives. Only HTMLElement and SVGElement are supported.`
+                );
+            }
+
+            // use "show" if the attribute is empty
+            const directives = parseDirectives(element.getAttribute(ErrorPlugin.errorAttribute) || 'show');
+
+            directives.forEach((directive) => {
+                if (directive.modifiers.length > 0) {
+                    // TODO: Use the `modifier.toString()` method once it's implemented
+                    throw new Error(
+                        `Modifiers are not supported for ${ErrorPlugin.errorAttribute} directives, but the following were found: "{${directive.modifiers.map((modifier) => `${modifier.name}: ${modifier.value}}`).join(', ')}" for element with tag "${element.nodeName}".`
+                    );
+                }
+            });
+
+            errorDirectives.push({
+                element,
+                directives,
+            });
+        });
+
+        return errorDirectives;
+    }
+
+    /**
+     * Parses the action and returns the opposite action if the element is not in an error state.
+     */
+    private parseErrorAction(action: string, isError: boolean) {
+        switch (action) {
+            case ErrorPlugin.supportedActions.show:
+                return isError ? 'show' : 'hide';
+
+            case ErrorPlugin.supportedActions.hide:
+                return isError ? 'hide' : 'show';
+
+            case ErrorPlugin.supportedActions.addClass:
+                return isError ? 'addClass' : 'removeClass';
+
+            case ErrorPlugin.supportedActions.removeClass:
+                return isError ? 'removeClass' : 'addClass';
+
+            case ErrorPlugin.supportedActions.addAttribute:
+                return isError ? 'addAttribute' : 'removeAttribute';
+
+            case ErrorPlugin.supportedActions.removeAttribute:
+                return isError ? 'removeAttribute' : 'addAttribute';
+
+            default:
+                throw new Error(
+                    `Unknown ${ErrorPlugin.errorAttribute} action "${action}". Supported actions are: ${Object.values(`"${ErrorPlugin.supportedActions}"`).join(', ')}.`
+                );
+        }
+    }
+
+    // Utils
+    private showElement(element: HTMLElement | SVGElement) {
+        element.style.display = 'revert';
+    }
+
+    private hideElement(element: HTMLElement | SVGElement) {
+        element.style.display = 'none';
+    }
```

Comment:
If the dev used `style="display: flex"` on element, what will happens when calling `showElement`? 

Can we add/remove a custom CSS class instead, like `data-live-hidden`? It would be safer than manipulating `element.style.display` IMO.

Reply:
Well, that sadly doesn't work, but this is a current issue with the `data-loading` attribute as well (see https://github.com/symfony/ux/issues/102).
A simple workaround is to do something like `class="hidden" data-live-error="addClass(flex) removeClass(hidden)"` instead. That will work.

I also don't think adding more custom CSS would be a good idea, as discussed above, and we would probably have to use `!important` any way to avoid the style not taking effect.